assets/src/blocks/Gallery/GalleryFrontend.js

```diff
@@ -0,0 +1,36 @@
+import { GalleryCarousel } from './GalleryCarousel';
+import { GalleryThreeColumns } from './GalleryThreeColumns';
+import { GalleryGrid } from './GalleryGrid';
+import { getGalleryLayout, GALLERY_BLOCK_CLASSES } from './getGalleryLayout';
+import { useGalleryImages } from './useGalleryImages';
+
+export const GalleryFrontend = ({
+  gallery_block_title,
+  gallery_block_description,
+  className,
+  gallery_block_style,
+  multiple_image,
+  gallery_block_focus_points
+}) => {
+  const layout = getGalleryLayout(className, gallery_block_style);
+  const postType = document.body.getAttribute('data-post-type');
+
+  const { images } = useGalleryImages({ multiple_image, gallery_block_focus_points }, layout);
+
+  return (
+    <section className={`block ${GALLERY_BLOCK_CLASSES[layout]}`}>
+      {gallery_block_title &&
+        <header>
+          <h2 className="page-section-header">{gallery_block_title}</h2>
```

Comment:
You have the same problem here we had with split 2 columns in https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/401 , I don't have any specific fix in mind (I also need to check how the RichText does its own escape at some point, there's an [`escapeAmpersand`](https://github.com/WordPress/gutenberg/blob/4857ad58c1241b3d63d21a6880c989b85746c3dc/packages/escape-html/src/index.js) over there), either RichText, dangerouslySetInnerHTML or other :shrug:

Reply:
I'm confused, I thought that if the component used `RichText` in the editor (which it does here) then it's not a problem? 🤔